mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/proto/DisableTaskExecutorsRequest.java

```diff
@@ -22,4 +22,6 @@
 @Value
 public class DisableTaskExecutorsRequest {
     Map<String, String> attributes;
+
+    Long expirationDurationInHours;
```

Comment:
Let's use Duration DS for this. Again, I want us to avoid capturing semantics in var names. 
``` suggestion
    Duration expirationDurationInHours;
```

Reply:
This is for the JSON payload in user request and specifying the time format for the duration here is just painful (plus the easy behavior regression from Jackson version etc.)

Reply:
I was thinking about this today, and I wanted to know if there was a reason why we want to have a configurable TTL per request. In other words, why not configure this at the control plane level?

Reply:
Yeah I was planning to configure this from the caller side (upgrade workflow) config (also inside nfmantis) as the length of this duration shall be decided by the upgrade workflow logic.